metricbeat/module/jolokia/jmx/_meta/data.json

```diff
@@ -1,33 +1,35 @@
 {
     "@timestamp": "2017-10-12T08:05:34.853Z",
-    "beat": {
-        "hostname": "host.example.com",
-        "name": "host.example.com"
+    "event": {
+        "dataset": "jolokia.jmx",
+        "duration": 115000,
+        "module": "jolokia"
     },
     "jolokia": {
-        "testnamespace": {
+        "jmx": {
+            "_namespace": "testnamespace",
```

Comment:
This shouldn't change, take a look to https://github.com/elastic/beats/pull/11103/files#diff-6e65049832821186bfacb7d0f748869cR102 for an example of migration with namespaces.

Reply:
Done, thanks!